### PR TITLE
derive Holder for tuple struct

### LINKED ISSUE #145: Generate Holder struct for it, support #[derive(Holder)] for tuple struct
This is the second milestone of #130.

### PR DESCRIPTION
**Pull request by ytanimura**
Close #145.

### CODE REVIEW

**Comment by ytanimura** [comment]
Generate Holder struct for it, support #[derive(Holder)] for tuple struct

File: ruststep-derive/src/entity.rs [hunk 1]
```diff
@@ -63,10 +63,7 @@ impl FieldEntries {
         let mut into_owned = Vec::new();
 
         for field in &st.fields {
-            let ident = field
-                .ident
-                .as_ref()
-                .expect_or_abort("Tuple struct case is not supported");
+            let ident = field.ident.as_ref().expect_or_abort("unreachable");
```
Comment (termoshtt) [inline ruststep-derive/src/entity.rs:66]: Do you know when it fails? If you know, please write here it.

File: ruststep-derive/src/lib.rs [hunk 2]
```diff
@@ -122,7 +123,11 @@ fn derive_holder(ast: &syn::DeriveInput) -> TokenStream2 {
     let attr = HolderAttr::parse(&ast.attrs);
     let ident = &ast.ident;
     match &ast.data {
-        syn::Data::Struct(st) => entity::derive_holder(ident, st, &attr),
+        syn::Data::Struct(st) => match st.fields {
+            syn::Fields::Named(_) => entity::derive_holder(ident, st, &attr),
+            syn::Fields::Unnamed(_) => type_decl::derive_holder(ident, st, &attr),
+            syn::Fields::Unit => panic!("Unit struct is not supported."),
```
Comment (termoshtt) [inline ruststep-derive/src/lib.rs:129]: Please use `abort_call_site!`

**Comment by ytanimura** [comment]
fixed them

**Pull request merged**